subsys/net/lib/wifi_ready/Kconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig WIFI_READY_EVENT_HANDLING
+	bool "WIFI ready event handling"
```

Comment:
Generic comment" Would `Wi-Fi status event handling` be more appropriate than `Wi-Fi ready event handling` ?

Reply:
I feel it is better to use `Wi-Fi ready events` as we have a separate Wi-Fi status mechanism that logs states like connected, disconnected, and scanning.

Reply:
Wouldn't that be `states` vs `status` ?